### ISSUE TITLE
"different format "-error with values in exponential notation

### ISSUE THREAD

**Issue by klaasman** [Popmotion/popmotion#423]
## Thrown error
```
Uncaught (in promise) Error: HEY, LISTEN!
Values '0px 1.5999999547489097e-8px 3.199999909497819e-8px rgba(161, 0, 246, 6.399999974426862e-10)'
and '0px 0px 0px rgba(161, 0, 246, 0)'
are of different format, or a value might have changed value type.
```

## Describe the bug
Popmotion itself generated those exponential numbers and then doesn't know how to handle them. The issue occured when I was rapidly toggling the pose.  
The configured poses were as follows:
```
{
  on: {
    boxShadow: '0px 0px 0px rgba(161, 0, 246, 0)',
  },
  off: {
    boxShadow: '0px 10px 20px rgba(0, 0, 0, 0.4)',
  },
}
```

## Device information
In which environment are you reproducing the bug?
 - Chrome 68

**Comment by InventingWithMonster** [comment]
To be clear, is this happening when you switch back between the two poses
mid-transition?

**Comment by klaasman** [comment]
That's correct.

**Comment by InventingWithMonster** [comment]
Fixed in `style-value-types@3.0.6`
